english/virtualreality/oculus.md

```diff
@@ -11,3 +11,5 @@ You have accidently entered into Kafka's world that has been redesigned
 by Murakami... you cannot escape it!
 
 [Life in the real world](problems/realproblems.md).
+
+You try to take off the oculus gear again to no avail.  You faintly hear Sonny and Cher's "I Got You Babe" in the background. After scanning the room you see Hansel smoking his joint. He's in the exact same position as he was yesterday.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.